src/app/models/build-log.ts

```diff
@@ -1,3 +0,0 @@
-export interface BuildLog {
```

Comment:
The discussion with @flynnduism at some point was to have typed logs and build logs, so that it's easier if the logs change (for example we add some new properties to the logs, like date time?).

For start, I think a simple `Message: string` property should eliminate our linter errors.